Libraries/LibWeb/CSS/Properties.json

```diff
@@ -2432,6 +2432,23 @@
       "repeat-y"
     ]
   },
+  "mask-size": {
+    "affects-layout": false,
+    "animation-type": "repeatable-list",
+    "inherited": false,
+    "initial": "auto",
+    "max-values": 2,
+    "valid-types": [
+      "length [0,∞]",
+      "percentage [0,∞]"
+    ],
+    "valid-identifiers": [
+      "auto",
+      "cover",
+      "contain"
+    ],
+    "percentages-resolve-to": "length"
```

Comment:
Not a change to make in this PR, but we could really do with `<bg-size>` being a properly supported type instead of having to specify all its requirements inside `Properties.json`.